### ISSUE TITLE
TaskConfiguration setter does not properly stringify TaskConfig

### ISSUE THREAD

**Issue by kenneth-marut-work** [eclipse-theia/theia#7956]
### Bug Description:
When trying to write a task configuration using `TaskConfigurationManger`'s `.setTaskConfigurations()` method, invalid JSON is written when stringifying ENUM types for `presentation.reveal` and `presentation.panel`.

### Steps to Reproduce:
1. Create an extension that injects the `TaskConfigurationManager`
2. Build an object of type `TaskConfiguration` with some values (fill in proper scope value), use enum types for presentation panel and reveal, for example:
```
        const pendingTaskConfiguration: TaskConfiguration = {
            label: 'my task',
            type: 'shell',
            command: 'top',
            problemMatcher: [],
            args: '',
            _scope: scopeURI,
            _source: scopeURI,
            options: {
                cwd: '',
            },
            presentation: {
                reveal: RevealKind.Always,
                panel: PanelKind.Shared
            },
        };
```
3. Attempt to call `this.taskConfigurationManager.setTaskConfigurations(scopeURI, [pendingTaskConfiguration])`
4. Observe that theia will throw a warning message `Invalid task configurations are found. Open tasks.json and find details in the Problems view.`
5. Open corresponding `tasks.json` file and observe that `reveal` and `panel` values have not been translated to their string equivalents

### Additional Information
![image](https://user-images.githubusercontent.com/62660714/83690629-ad661200-a5b6-11ea-9873-e1ae34b4ff70.png)

- Operating System:
Ubuntu
- Theia Version:
1.2

**Comment by RomanNikitenko** [comment]
@kenneth-marut-work 
is it fixed within https://github.com/eclipse-theia/theia/pull/7982?
can we close the issue?

**Comment by kenneth-marut-work** [comment]
Yes, this is an old issue addressed by #7982. I'll go ahead and close this
